tests/parsers/beckman_pharmspec/testdata/hiac_example_1.json

```diff
@@ -0,0 +1,371 @@
+                                                            },
+                                                            "cumulative count": {
+                                                                "value": 0.0,
+                                                                "unit": "(unitless)"
+                                                            },
+                                                            "cumulative particle density": {
+                                                                "value": 90.0,
+                                                                "unit": "Counts/mL"
+                                                            }
+                                                        },
+                                                        {
+                                                            "particle size": {
+                                                                "value": 50.0,
+                                                                "unit": "μm"
+                                                            },
+                                                            "cumulative count": {
+                                                                "value": 0.0,
+                                                                "unit": "(unitless)"
+                                                            },
+                                                            "cumulative particle density": {
+                                                                "value": 100.0,
+                                                                "unit": "Counts/mL"
+                                                            }
+                                                        }
+                                                    ]
+                                                }
+                                            ]
+                                        }
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            }
+        ],
+        "data system document": {
+            "file name": "hiac_example_1.xlsx",
+            "software name": "PharmSpec",
+            "software version": "3.0",
+            "ASM converter name": "allotropy",
+            "ASM converter version": "0.1.23"
+        },
+        "device system document": {
+            "equipment serial number": "1808303021",
+            "device document": [
+                {
+                    "detector identifier": "",
+                    "detector model number": "HRLD150@10ml"
+                }
+            ]
+        },
+        "calculated data aggregate document": {
+            "calculated data document": [
+                {
+                    "calculated data name": "Average",
```

Comment:
Typically, calculated data documents have a "calculated data identifier" (a UUID) and a data source document that link them back to the measurements they are derived from.

@LimulusPolyphemus thoughts?

Reply:
@chrisfriedline-benchling - @nathan-stender is correct, the "calculated data identifier" should always be included so the "calculated result" can be referenced elsewhere if needed.

Reply:
@chrisfriedline-benchling looking at this again - the `calculate data documents` should also include the `data source aggregate document` with a `data source documents` with the `data source identifier` field to cite the `measurement identifier` from which the input data for the Average is derived.